testing/logstash/logstash.post-deinstall

```diff
@@ -1,5 +1,6 @@
 #!/bin/sh
 
 deluser logstash 2>/dev/null
+rm -f /etc/profile.d/logstash 2>/dev/null
```

Comment:
What the... Maybe you want also remove whole home directory as well? Just for fun, users love when pkgs removes their files when uninstalling...